### DISCUSSION TITLE
Nothing is working anymore...

### DISCUSSION THREAD

**Discussion by Leonie22** [anonaddy/anonaddy#259] [Q&A]
Hi there,
I've just paid my bill for this year with a Pro Plan but my aliases are not working anymore...
Eveything was working successfully. I forgot to renew my plan for a week so my aliases were not  working. 3 days ago I paid the invoice but it's still not working  :-(
Any idea ? Thanks for your help and your time.
Leo

**Comment by willbrowningme** [comment]
Hi there,

If you allowed your subscription to expire then your account will have been downgraded to the Free plan.

https://anonaddy.com/faq/#what-happens-if-i-have-a-subscription-but-then-cancel-it

This means if you used any aliases with paid plan only domains they will have been deactivated.

You will need to activate any aliases that you wish to use.

If you are still having issues please contact me.

**Reply by Leonie22** [reply]
Ok sorry, it seems it's because I didn't renew my domain... sorry for the inconvenience
